src/concepts/choosing_a_dataset_mobile.md

```diff
@@ -0,0 +1,96 @@
+# Choosing a Dataset - Mobile Edition
+
+## Products Overview 
```

Comment:
Our spell check CI is throwing a bunch of errors (copied below). Most of these are technical terms that should be wrapped with backticks. The product names are exceptions and should be added to the .spelling file. [Docs for reference](https://docs.telemetry.mozilla.org/meta/contributing.html#spell-checking).


```
$ mdspell 'src/**/*.md' --ignore-numbers --en-us --report
    src/concepts/choosing_a_dataset_mobile.md
       14 | | Klar                   | `Klar`    
       22 | Klar is the release name for Focus 
       26 | ome telemetry is also sent by FirefoxReality and some non-Mozilla forks of 
       33 |  by Focus Android, Focus iOS, Klar, Firefox for FireTV, Firefox  
       33 |  Focus iOS, Klar, Firefox for FireTV, Firefox for Echo Show and Fi 
       37 | s/Telemetry.md), [Firefox for FireTV](https://github.com/mozilla-m 
       50 | cs included in metadata are: [document_id, timestamp, date, geo_country 
       50 | document_id, timestamp, date, geo_country, geo_city, app_build_id, norm 
       50 | timestamp, date, geo_country, geo_city, app_build_id, normalized_cha 
       50 |  date, geo_country, geo_city, app_build_id, normalized_channel] as descr 
       50 | ntry, geo_city, app_build_id, normalized_channel] as described [here](https:// 
       59 | lients`) are derived from the saved_session ping only available as an opt 
       64 |  from Focus (iOS, Android and Klar) get sent to `telemetry_focus 
       73 |  table, including Lockbox and FirefoxReality.  For more information on the 
       78 |  Focus Android, Focus iOS and Klar. 
       80 | ng `app_build_id`(s) for that app_version.  This can be found by reachi 
       86 | pment.  Users sign up to test pre-released versions of the app.  In Focu 
       86 |  `beta` channel uses the same APK in the Google Play Store as t 
       86 | ing back and flagging them by clientid). Beta releases have their `n 
       88 | mation see the blameless post mortem document [here](https://docs. 
       92 | new SDK starting with Project Fenix, then update the other existi 
       92 | isting apps to Glean starting H22019. Instead of `core` and `event 
```

Reply:
should be all fixed now